pyproject.toml

```diff
@@ -10,7 +10,7 @@ description = "A pure-python CLI application to manipulate PDF files"
 readme = "README.md"
 dynamic = ["version"]
 license = { file = "LICENSE" }
-requires-python = ">=3.7.0"
+requires-python = ">=3.8.0"
```

Comment:
There are a few other places where this change should be made:
* remove the `3.7` classifier on line 26 of this file
* remove the version of `3.7` in `docs/user/installation.md`